manifests/windows/install.pp

```diff
@@ -12,8 +12,17 @@
     default   => $::puppet_agent::arch
   }
 
+  if $::puppet_agent::is_pe {
```

Comment:
@ferventcoder I think this line is handling that, albeit in a somewhat roundabout way: https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/master/manifests/params.pp#L13